File: src/Entities/Command.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

import java.sql.Date;

/**
 *
 * @author Shayma CHAIBI
 */
public class Command {

    
    private int IdCmd;
    private int IdProd;
    private int client_id;
    private String DesCmd;
    private Date CmdDate;
    private Date DlvrDate;
    private float CmdTotal;
    private String CmdStatus;
    private String DlvrStatus;
    
    public Command() {
        super();
        //throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
    }
   
    
    public Command(int IdCmd, int IdProd, int client_id, String DesCmd, Date CmdDate, Date DlvrDate, float CmdTotal, String CmdStatus, String DlvrStatus) {
        this.IdCmd = IdCmd;
        this.IdProd = IdProd;
        this.client_id = client_id;
        this.DesCmd = DesCmd;
        this.CmdDate = CmdDate;
        this.DlvrDate = DlvrDate;
        this.CmdTotal = CmdTotal;
        this.CmdStatus = CmdStatus;
        this.DlvrStatus = DlvrStatus;
    }

    public int getIdCmd() {
        return IdCmd;
    }

    public int getIdProd() {
        return IdProd;
    }

    public int getClient_id() {
        return client_id;
    }

    public String getDesCmd() {
        return DesCmd;
    }

    public Date getCmdDate() {
        return CmdDate;
    }

    public Date getDlvrDate() {
        return DlvrDate;
    }

    public float getCmdTotal() {
        return CmdTotal;
    }

    public String getCmdStatus() {
        return CmdStatus;
    }

    public String getDlvrStatus() {
        return DlvrStatus;
    }
    
    public void setIdCmd(int IdCmd) {
        this.IdCmd = IdCmd;
    }

    public void setIdProd(int IdProd) {
        this.IdProd = IdProd;
    }

    public void setClient_id(int client_id) {
        this.client_id = client_id;
    }

    public void setDesCmd(String DesCmd) {
        this.DesCmd = DesCmd;
    }

    public void setCmdDate(Date CmdDate) {
        this.CmdDate = CmdDate;
    }

    public void setDlvrDate(Date DlvrDate) {
        this.DlvrDate = DlvrDate;
    }

    public void setCmdTotal(float CmdTotal) {
        this.CmdTotal = CmdTotal;
    }

    public void setCmdStatus(String CmdStatus) {
        this.CmdStatus = CmdStatus;
    }

    public void setDlvrStatus(String DlvrStatus) {
        this.DlvrStatus = DlvrStatus;
    }

    public void setName(String string) {
        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
    }

}

File: BODYUP1/src/Entity/Command.php
<?php

namespace App\Entity;

use App\Repository\CommandRepository;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity(repositoryClass=CommandRepository::class)
 */
class Command
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="integer")
     */
    private $IdCmd;

    /**
     * @ORM\Column(type="integer")
     */
    private $IdProd;

    /**
     * @ORM\Column(type="integer")
     */
    private $IdClient;

    /**
     * @ORM\Column(type="string", length=200, nullable=true)
     */
    private $DesCmd;

    /**
     * @ORM\Column(type="date")
     */
    private $CmdDate;

    /**
     * @ORM\Column(type="date")
     */
    private $DlvrDate;

    /**
     * @ORM\Column(type="float")
     */
    private $TotalCmd;

    /**
     * @ORM\Column(type="string", length=100, nullable=true)
     */
    private $CmdStatus;

    /**
     * @ORM\Column(type="string", length=100, nullable=true)
     */
    private $DlvrStatus;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getIdCmd(): ?int
    {
        return $this->IdCmd;
    }

    public function setIdCmd(int $IdCmd): self
    {
        $this->IdCmd = $IdCmd;

        return $this;
    }

    public function getIdProd(): ?int
    {
        return $this->IdProd;
    }

    public function setIdProd(int $IdProd): self
    {
        $this->IdProd = $IdProd;

        return $this;
    }

    public function getIdClient(): ?int
    {
        return $this->IdClient;
    }

    public function setIdClient(int $IdClient): self
    {
        $this->IdClient = $IdClient;

        return $this;
    }

    public function getDesCmd(): ?string
    {
        return $this->DesCmd;
    }

    public function setDesCmd(?string $DesCmd): self
    {
        $this->DesCmd = $DesCmd;

        return $this;
    }

    public function getCmdDate(): ?\DateTimeInterface
    {
        return $this->CmdDate;
    }

    public function setCmdDate(\DateTimeInterface $CmdDate): self
    {
        $this->CmdDate = $CmdDate;

        return $this;
    }

    public function getDlvrDate(): ?\DateTimeInterface
    {
        return $this->DlvrDate;
    }

    public function setDlvrDate(\DateTimeInterface $DlvrDate): self
    {
        $this->DlvrDate = $DlvrDate;

        return $this;
    }

    public function getTotalCmd(): ?float
    {
        return $this->TotalCmd;
    }

    public function setTotalCmd(float $TotalCmd): self
    {
        $this->TotalCmd = $TotalCmd;

        return $this;
    }

    public function getCmdStatus(): ?string
    {
        return $this->CmdStatus;
    }

    public function setCmdStatus(?string $CmdStatus): self
    {
        $this->CmdStatus = $CmdStatus;

        return $this;
    }

    public function getDlvrStatus(): ?string
    {
        return $this->DlvrStatus;
    }

    public function setDlvrStatus(?string $DlvrStatus): self
    {
        $this->DlvrStatus = $DlvrStatus;

        return $this;
    }
}
